app/build.gradle

```
@@ -34,8 +34,7 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.6.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.6.0'
```

También la última versión del conversor
```suggestion
    implementation 'com.squareup.retrofit2:converter-gson:2.6.2'
```